tests/test_forcefields.py

```diff
@@ -84,3 +84,51 @@ def test_load_default():
     for (ref_handle, test_handle) in zip(ref.get_ordered_handles(), test.get_ordered_handles()):
         assert ref_handle.smirks == test_handle.smirks
         np.testing.assert_array_equal(ref_handle.params, test_handle.params)
+
+
+def test_split():
+    ffs = load_split_forcefields()
```

Comment:
Test expansion request: Would it be informative also to assert self-consistency with the order of handles assumed by `parameterize_nonbonded` methods in `fe` module?

Reply:
The parameters are referenced by name (https://github.com/proteneer/timemachine/blob/master/timemachine/fe/free_energy.py#L196), unless you were thinking of something else.

Reply:
Ahh, sorry, I see. I guess it's possible to pass handles in a wrong order to 
https://github.com/proteneer/timemachine/blob/0628c0e8465805c94bad0d70d25a0abbde3d522e/timemachine/fe/free_energy.py#L196-L203

but that wouldn't have anything to do with consistency of `ff_params` and `topology.parameterize_nonbonded`.

Don't have great ideas for an integration test here -- and maybe this is already tested thoroughly enough? -- but some test to check that the generated `sys_params` (not just the `ff_params`) are correct in the presence of IG splitting may be helpful.

Reply:
Right. Not sure either. Will merge but think about refactoring this later.